docs/structure/configure.md

```diff
@@ -0,0 +1,114 @@
+(structure/configure)=
+# Configuration options for the table of contents
+
+This page covers some of the options you have available to control your book's behavior via the Table of Contents.
+
+:::{seealso}
+For an overview of the configuration structure for the Table of Contents, see [](toc/configuration-structure)
+:::
+
+## Add captions to parts
+
+To add a caption to a Part (so that it shows up in the sidebar, for example) use the `caption:` option like so:
+
+```
+- caption: My part name
+  chapters:
+  - file: chapter1
+  ...
+```
+
+## Specify alternate titles
+
+If you'd like to specify an alternate title from the one defined within a file,
+you may do so with the `title:` key. For example:
+
+```yaml
+- file: path/to/myfile
+  title: My alternate page title
+```
+
+Note that this only applies to the sidebar
+in the table of contents, it does not change the actual chapter/section title.
+
+(toc/numbering)=
+## Number your chapters and sections
+
+You can automatically add numbers to each chapter of your book.
+To add numbers to **all chapters of your book**, add the `numbered: true` flag to your book's defaults, like so:
+
+```yaml
+format: jb-book
```

Comment:
you should mention `jb-article` as well

Reply:
In the latest commit I've added a mention of `jb-article`, lemme know how that looks